src/views/register/styles.js

```diff
@@ -0,0 +1,10 @@
+import { makeStyles } from '@mui/styles';
```

Comment:
need talk about the dependency....

Reply:
Hey @winny1212, the jsx structure seems good!!!
The `@mui/styles` is depreciated, and no longer used in v5. Have a look at this https://mui.com/styles/basics/#main-content.
What was it used for?